src/components/Restoration/CorrectNameRequest.vue

```diff
@@ -1,59 +1,58 @@
 <template>
-  <v-form id="correct-nr-form" ref="correctNrForm" v-model="formValid" lazy-validation>
-    <!-- Dialogs -->
+  <v-form id="correct-name-request-form" ref="form" v-model="formValid" lazy-validation>
     <ConfirmDialogShared
       ref="confirm"
       attach="#app"
     />
 
     <v-row no-gutters>
-      <v-col cols="1" class="mt-1">
-        <v-btn x-small fab outlined :ripple="false" color="gray7" class="step-icon" tabindex="-1">1</v-btn>
+      <v-col cols="1" class="mt-3">
+        <v-chip outlined class="step-icon">1</v-chip>
```

Comment:
Before:
![image](https://user-images.githubusercontent.com/10002889/219828830-5f609af3-839d-4be4-8137-2bac70115276.png)

After:
![image](https://user-images.githubusercontent.com/10002889/219828814-3bf5fa14-7283-469c-bd71-cc16fa8df7cf.png)

V-chip ftw!

Reply:
This component is now in BCRS shared repo.